docs/Multilang-protocol.md

```diff
@@ -7,6 +7,18 @@ This page explains the multilang protocol as of Storm 0.7.1. Versions prior to 0
 
 # Storm Multi-Language Protocol
 
+## Supported Lanugages
+
+Storm Multi-Language has implementation in the following languages:
+
+- [JavaScript](../storm-multilang/javascript)
```

Comment:
The docs will be compiled as html and placed to website, so docs should not refer outside of docs directory. You can describe maven artifacts instead, like
* Javascript: org.apache.storm / multilang-javascript
* Python: org.apache.storm / multilang-python
* Ruby: org.apache.storm / multilang-ruby

Reply:
got it - fixed, thanks